app/src/components/GrantCard.vue

```diff
@@ -24,13 +24,15 @@
 
       <!--cart button-->
       <div class="absolute bottom-0 right-0">
-        <button
-          class="btn opacity-100 md:opacity-0 group-hover:opacity-100"
-          :class="{ 'in-cart': isInCart(id) }"
-          @click.stop="isInCart(id) ? removeFromCart(id) : addToCart(id)"
-        >
-          <CartIcon />
-        </button>
+        <template v-if="idList.includes(id)">
```

Comment:
Since `idList` can be undefined after removing the unneeded computed, just make sure to account for that. I think either of these should work, but worth testing
```suggestion
        <template v-if="idList?.includes(id)">
```

```suggestion
        <template v-if="idList && idList.includes(id)">
```